src/modules/returns/lib/csv-adapter/validator.js

```diff
@@ -169,7 +169,7 @@ const validateReturnReference = licence => {
 
 const isValidBooleanOrEmpty = (licence, licenceKey, errorKey = licenceKey) => {
   const { value, line } = licence[licenceKey];
-  const isValid = ['y', 'n', ''].includes(value.toLowerCase());
+  const isValid = ['y', 'n', 'yes', 'no', ''].includes(value.toLowerCase());
```

Comment:
Is it worth changing `value.toLowerCase()` to `value.toLowerCase().trim()`?

Reply:
Oooh, yes. Thanks for that. Updated :)